### ISSUE TITLE
Fail to parse ABNAMRO transaction with currency exchange.

### ISSUE THREAD

**Issue by dmitry-mukhin** [felgru/bank-statement-parser#1]
both BEA patterns don't match if transaction includes currency exchange data.
e.g.:

```
BEA, Google Pay
store,card
NR:xxx date/time
DRNIS,Land: HR
HRK 44,00 1EUR=7,4074074 HRK
ECB Koers=7,5620027 OPSLAG 2,09%
KOSTEN •0,15 ACHTERAF BEREKEND
```

**Comment by felgru** [comment]
Thank you for the detailed description of the transaction that didn't parse.
I have changed the parser for BEA transactions, so that it should now be able to parse your transaction without any errors.

Can you retry with the latest version from the master branch?

**Comment by felgru** [comment]
I'm closing this issue since I assume that it has been fixed with commit 3305b49844dc40bc60721d26d3ebac110c11ca00.

**Comment by dmitry-mukhin** [comment]
sorry i didn't have time to check this in 2022 :)
thanks for addressing this anyway!
